src/muya/lib/assets/styles/index.css

```diff
@@ -97,6 +97,25 @@ div.ag-show-quick-insert-hint p.ag-paragraph.ag-active > span.ag-paragraph-conte
   word-break: break-word;
 }
 
+/* TODO: The current implementation wraps long lines in code blocks
```

Comment:
The design idea of Mark Text is that what you type and what you see is also a preview at the same time when you edit, so we don’t really want to use a style when editing, and a different style when previewing, so I still prefer that no-wrap in edit mode and preview status. After waiting for other contributors to add user settings, choose whether to wrap code block lines.

Reply:
I pushed an update.